back/api/models.py

```diff
@@ -40,14 +39,17 @@ class Meta:
         verbose_name_plural = _('order types')
 
 
-class Identity(User):
+class Identity(AbstractUser):
+    """
+    Extends User model. All Identities are users but all the users can login.
+    """
     street = models.CharField(_('street'), max_length=100, blank=True)
     street2 = models.CharField(_('street2'), max_length=100, blank=True)
     postcode = models.PositiveIntegerField(_('postcode'), blank=True, null=True)
     city = models.CharField(_('city'), max_length=50, blank=True)
     country = models.CharField(_('country'), max_length=50, blank=True)
     company_name = models.CharField(_('company_name'), max_length=50, blank=True)
-    phone = models.TextField(_('phone'), blank=True)
+    phone = models.CharField(_('name'), max_length=50, blank=True)
```

Comment:
Shouldn't that be `phone` for the translation string ?

Reply:
@kalbermattenm 
oh, yeah you're right, are you ok to review my PR's? :)

Reply:
That's what I did... ;-)